packages/tailwindcss/src/functions.test.ts

```diff
@@ -285,6 +285,20 @@ describe('theme function', () => {
             }"
           `)
         })
+
+        test('theme(fontFamily.sans)', async () => {
+          expect(
+            await compileCss(css`
+              .fam {
+                font-family: theme(fontFamily.sans);
+              }
+            `),
+          ).toMatchInlineSnapshot(`
+          ".fam {
+            font-family: ui-sans-serif, system-ui, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+          }"
+        `)
+        })
       })
```

Comment:
This test is for #14262 but it requires the changes in here (the default theme changes) to be reproducible so we didn't cherry pick it into the other PR. I can make yet-another-PR if request but I think it's reasonable to add this here since it does also test that the default theme works for the CSS `theme()` function